python/ambassador/cache.py

```diff
@@ -242,7 +242,6 @@ def __init__(self, logger: logging.Logger) -> None:
         self.logger = logger
         self.logger.debug("NullCache: INIT")
         self.reset_stats()
-        pass
```

Comment:
was this manually removed or from a linter? seems slightly odd for an import checker to remove. Maybe autoflake?

Reply:
yea `pass` is a no-op in python. it basically tells the python interpreter to do nothing